litmus-portal/frontend/src/components/UserManagementTab/UserManagement/index.tsx

```diff
@@ -0,0 +1,275 @@
+import {
+  createStyles,
+  FormControl,
+  IconButton,
+  InputAdornment,
+  Paper,
+  Select,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TextField,
+  Theme,
+  Toolbar,
+  Typography,
+  withStyles,
+} from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import React from 'react';
+import bin from '../../../assets/icons/bin.svg';
+import user from '../../../assets/icons/user.svg';
+import ButtonFilled from '../../Button/ButtonFilled';
+import CreateUser from '../CreateUser';
+import useStyles from './styles';
+
+// StyledTableCell used to create custom table cell
+const StyledTableCell = withStyles((theme: Theme) =>
+  createStyles({
+    head: {
+      backgroundColor: theme.palette.common.white,
+      color: theme.palette.common.black,
+    },
+    body: {
+      fontSize: 14,
+    },
+  })
+)(TableCell);
+
+// data for each user name
+interface User {
+  status: boolean;
+  name: string;
+}
+
+// data for each user
+interface Row {
+  Name: User;
+  username: string;
+  email: string;
+  userCreated: string;
+}
+
+// UserManagement displays users table
+const UserManagement: React.FC = () => {
+  const classes = useStyles();
+  const [showDiv, setShowDiv] = React.useState<boolean>(false);
+
+  // dummy data for now
+  let selRow = [];
+  selRow = [
```

Comment:
Instead of creating it above and using it in useState directly write it there, like:

```
const [rows, setRows] = React.useState<Row[]>([
{ data },
{ other Data }
]);
```

Reply:
This data is used in 2 useSates, using it twice will increase the length of the code. Also, I will be removing this dummy since data will be fetched from backend.